cmd/machine-controller/main.go

```diff
@@ -163,6 +167,7 @@ func main() {
 	flag.StringVar(&nodePauseImage, "node-pause-image", "", "Image for the pause container including tag. If not set, the kubelet default will be used: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/")
 	flag.StringVar(&nodeKubeletRepository, "node-kubelet-repository", "quay.io/kubermatic/kubelet", "Repository for the kubelet container. Only has effect on Flatcar Linux.")
 	flag.StringVar(&nodeContainerRuntime, "node-container-runtime", "docker", "container-runtime to deploy")
+	flag.Var(&nodeContainerdRegistryMirrors, "node-containerd-registry-mirrors", "Configure registry mirrors endpoints. Can be used multiple times to specify multiple mirrors")
```

Comment:
It seems I can't add a comment to the line for `node-registry-mirrors` above this one, but should we update the description for `node-registry-mirrors` to either call it deprecated or add that it only is effective for `docker.io` images?

Reply:
it's not depricated :)

Reply:
Gotcha, wasn't sure what status we considered for it.